x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/plan/logical/UnaryPlan.java

```diff
@@ -42,6 +43,11 @@ public List<Attribute> output() {
         return child.output();
     }
 
+    @Override
+    public AttributeSet outputSet() {
+        return child().outputSet();
+    }
+
```

Comment:
I'm afraid this is incorrect: now all the subplans that override `output()` but not `outputSet()` (eg. `Eval`) return a wrong output set, based on the child rather than on the actual output